Automattic/wp-calypso
# Staging Sites: Add Staging Sites UI in Hosting Configuration

Related to 1696-gh-Automattic/dotcom-forge

## Proposed Changes

In this PR, I propose to add an initial UI that allows adding a staging site.

## Testing Instructions



1. Open the Atomic site
2. Navigate to Hosting Configuration
3. Confirm that the "Staging site" card is displayed
4. Click "Create staging site" to add a new staging site
5. Confirm that when the site is created, the UI changes and allows navigating to staging site

<img width="705" alt="Screenshot 2023-02-23 at 14 35 33" src="https://user-images.githubusercontent.com/727413/220922814-a5cf0c7d-397c-4ff3-bffb-bb1bffa7155e.png">

<img width="705" alt="Screenshot 2023-02-23 at 14 34 41" src="https://user-images.githubusercontent.com/727413/220922829-7ca54c1f-9753-4fbb-8926-ef0ce60bc2c8.png">


## Pre-merge Checklist
